### PR TITLE
Feature/Handling VR help requests when application does not send VR help prompt

### PR DESCRIPTION
**Pull request by KhrystynaDubovyk**
Update TTS, UI.SetGlobalProreties, VR.AddCommand according to the proposal

### CODE REVIEW

**Comment by AStasiuk** [comment]
@mrapitis, @robink this PR is ready for review

File: docs/UI/SetGlobalProperties/index.md [hunk 1]
```diff
@@ -14,16 +14,14 @@ SDL requests to set-up the data for VR help layout, the name and icon for in-app
 
 The request may arrive for the application whatever being active or in background on HMI (depends on Policy Table permissions applicable to mobile application request, by default allowed to operate in all HMI levels except of NONE).
```
Comment (mrapitis) [inline docs/UI/SetGlobalProperties/index.md]: whatever should be whether

File: docs/TTS/SetGlobalProperties/index.md [hunk 2]
```diff
@@ -10,8 +10,9 @@ Purpose
 : Set the properties for the TTS component.
 
 ### Request
+On receiving `AddCommand` with `CommandType = Command` SDL must send updated values of `helpPrompt` via UI.SetGlobalProperties request to HMI.
 
-#### Parameters
+#### Parameters
```
Comment (mrapitis) [inline docs/TTS/SetGlobalProperties/index.md]: extra space introduced

**Comment by KhrystynaDubovyk** [comment]
@mrapitis, @robinmk, this PR is ready for review.
Please, suggest your changes.

**Comment by KhrystynaDubovyk** [comment]
@mrapitis, @robinmk 
Did you have a chance to look over the PR? Do you have any comments?

**Pull request merged**